docs/integration.rst

```diff
@@ -695,18 +703,6 @@ Cloud Translate Text Operators
     Translate a string or list of strings.
 
 
-Cloud Video Intelligence
```

Comment:
Why this removed?

Reply:
Thanks for catching this. 
I'm not sure what happened. 
I reforked from master made my additions must have accidentally removed somethings.